### DISCUSSION TITLE
Is KeePassXC Blocking Me From Manually Entering a Password?

### DISCUSSION THREAD

**Discussion by Clueless-in-Seattle** [keepassxreboot/keepassxc#6602] [Q&A] [answered]
A contributor to a local BBS suggested I install KeePassXC on my refurbished Dell Latitude 3330. So I downloaded a file named KeePassXC-2.6.4-Win64.msi and installed it. But I never got around to creating a database of my passwords because other problems kept cropping up and taking up my time.

But I noticed that KeePassXC would automatically run at bootup, in spite of it not having any passwords entered into its database.

Today, I tried to log into my Social Security account, where they now have a two step process where you enter your user I.D. and password, and then they email you an 8 digit security code you have to enter on the SS login page. But the field where I was supposed to enter the security code was partially covered up by a little doodad like this:

![KeePassXC Fill TOIP Blocks Entry of Security Code w Arrow](https://user-images.githubusercontent.com/42583333/121240740-85eff800-c84f-11eb-92f0-e698bf4373f5.jpg)

It looked like a tiny gray message balloon containing three dots in a horizontal row. And when I hovered my mouse pointer over it, a box appeared containing the words "Fill TOTP from KeePassXC." The problem was that that little gray balloon covered up the the rightmost end of the field where the 8th digit of the security code was supposed to go, and seemed to be blocking me from entering the 8th digit. I could enter the first 7 digits, but I couldn't enter the last digit where the grey balloon was squatting. And I was denied access to my account.

So, I thought, "I don't have any passwords in KeePass yet, so I'll just uninstall it, and then try accessing my SS account." So that's what I did; I uninstalled KeePassXC, using Windows 10 add/remove programs, but when I retried accessing my SS account, that same little "doodad" still appeared in the same position, blocking me from entering the last digit of the 8 digit security code. I tried shutting down and restarting Windows, but the little sucker was still there after the reboot.

I'd appreciate some help figuring out why I'm getting that KeePassXC symbol and message even after I've uninstalled it. And how I can get rid of it.

_Will in Seattle
a.k.a. "Clueless"_ 

BTW, the same little gray doodad appeared just now when I logged in here a github.

**Comment by droidmonkey** [comment, marked as answer]
Remove or disable the keepassxc browser extension in your browser.

**Reply by Clueless-in-Seattle** [reply]
Bingo!  That did the trick!  Many thanks!
